x-pack/test/functional/apps/transform/cloning.ts

```diff
@@ -46,21 +47,182 @@ export default function ({ getService }: FtrProviderContext) {
 
     const testDataList = [
       {
-        suiteTitle: 'batch transform with terms group and avg agg',
-        expected: {},
+        suiteTitle: 'clone transform',
+        transformId: `clone_${transformConfig.id}`,
+        transformDescription: `a cloned transform`,
+        get destinationIndex(): string {
+          return `user-${this.transformId}`;
+        },
+        expected: {
+          aggs: {
+            index: 0,
+            label: 'products.base_price.avg',
+          },
+          indexPreview: {
+            columns: 10,
+            rows: 5,
+          },
+          groupBy: {
+            index: 0,
+            label: 'category',
+          },
+          pivotPreview: {
+            column: 0,
+            values: [
+              `Men's Accessories`,
+              `Men's Clothing`,
+              `Men's Shoes`,
+              `Women's Accessories`,
+              `Women's Clothing`,
+            ],
+          },
+        },
       },
     ];
 
     for (const testData of testDataList) {
       describe(`${testData.suiteTitle}`, function () {
         after(async () => {
-          // await transform.api.deleteIndices(<CLONE_DEST_INDEX>);
+          await transform.api.deleteIndices(testData.destinationIndex);
```

Comment:
We should also cleanup the created index pattern for the destination index:
```
await transform.testResources.deleteIndexPatternByTitle(testData.destinationIndex);
```

Reply:
Updated in 3a20fc4.